dracut/30ignition/ignition-generator

```diff
@@ -37,7 +38,7 @@ add_requires() {
 # boot.
 if $(cmdline_bool coreos.first_boot 0); then
     add_requires ignition-disks.service
-    add_requires ignition-files.service
+    add_requires ignition-files.service initrd-switch-root.service
```

Comment:
should one of these be a target considering we are modifying add_requires to accept $2 as `target=` ?

Reply:
can we add a comment in here to that effect ?